testing/integration/install_test.go

```diff
@@ -32,34 +31,16 @@ func TestInstall(t *testing.T) {
 		// It's not safe to run this test locally as it
 		// installs Elastic Agent.
 		Local: false,
-
-		// Since this test sets the location on the
-		// filesystem where Elastic Agent will be
-		// installed, it's probably safest not to run
-		// it along with other tests.
-		Isolate: true,
```

Comment:
We don't isolate this test anymore even if we install the agent ?

Reply:
The fixture handles proper cleanup, so there is no need for it to be isolated.